compiler/passes/expandExternArrayCalls.cpp

```diff
@@ -97,6 +100,14 @@ void expandExternArrayCalls() {
 
     if (fcopy) {
       SET_LINENO(fn);
+      Expr* parentScope = fn->defPoint->parentExpr;
```

Comment:
My initial reaction was that this should be parent symbol and not expr, but I see that this'll have a tighter scope. A defPoint for an FnSymbol can't exist where parentScope could get something funny can it?

Reply:
I believe I am definitely assuming that any scope which can contain an `extern proc` can contain a `use`.  And I can't think of a counterexample offhand.